libtransmission/rpcimpl.cc

```diff
@@ -1013,6 +1026,11 @@ char const* torrentSet(tr_session* session, tr_variant::Map const& args_in, tr_v
             tor->set_sequential_download(*val);
         }
 
+        if (auto const val = args_in.value_if<int64_t>(TR_KEY_sequential_download_from_piece); errmsg == nullptr)
```

Comment:
Looks like this is missing a check.

```suggestion
        if (auto const val = args_in.value_if<int64_t>(TR_KEY_sequential_download_from_piece); val && errmsg == nullptr)
```